src/Goerli-DssSpell.t.base.sol

```diff
@@ -1561,4 +1568,23 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
     function checkChainlogVersion(string memory key) internal {
         assertEq(chainLog.version(), key, concat("TestError/Chainlog-version-mismatch-", key));
     }
+
+    function checkRWADocUpdate(bytes32 ilk, string memory currentDoc, string memory newDoc) internal {
+        (string memory doc, address pip, uint48 tau, uint48 toc) =
```

Comment:
No need for a line break in the middle of this assignment (in the rest of the file long lines are preferred).